release/sdk-config.yaml.tmpl

```diff
@@ -37,15 +37,15 @@ commands:
 - name: sandbox
   path: daml-helper/daml-helper
   desc: "Launch the Sandbox"
-  args: ["run-jar", "daml-sdk/daml-sdk.jar", "sandbox"]
+  args: ["run-jar", "--logback-config=daml-sdk/sandbox-logback.xml", "daml-sdk/daml-sdk.jar", "sandbox"]
```

Comment:
LGTM, but we should pass the same argument when running this from `daml-helper` (in `daml start`). Happy to do this in a separate PR though.